tests/testthat/recorded_tests/app-waitForValue/tests/shinytest/mytest-expected/001.json

```diff
@@ -40,48 +40,45 @@
       }
     },
     "dynamic_output": {
-      "html": "<div class=\"form-group shiny-input-container\">\n  <label class=\"control-label\" id=\"bins-label\" for=\"bins\">Number of bins:<\/label>\n  <input class=\"js-range-slider\" id=\"bins\" data-skin=\"shiny\" data-min=\"1\" data-max=\"50\" data-from=\"30\" data-step=\"1\" data-grid=\"true\" data-grid-num=\"9.8\" data-grid-snap=\"false\" data-prettify-separator=\",\" data-prettify-enabled=\"true\" data-keyboard=\"true\" data-data-type=\"number\"/>\n<\/div>\n<div id=\"distPlot\" class=\"shiny-plot-output\" style=\"width:100%;height:400px;\"><\/div>",
+      "html": "<div class=\"form-group shiny-input-container\">\n  <label class=\"control-label\" id=\"bins-label\" for=\"bins\">Number of bins:<\/label>\n  <input class=\"js-range-slider\" id=\"bins\" data-skin=\"shiny\" data-min=\"1\" data-max=\"50\" data-from=\"30\" data-step=\"1\" data-grid=\"true\" data-grid-num=\"9.8\" data-grid-snap=\"false\" data-prettify-separator=\",\" data-prettify-enabled=\"true\" data-keyboard=\"true\" data-data-type=\"number\"/>\n<\/div>\n<div class=\"shiny-plot-output html-fill-item\" id=\"distPlot\" style=\"width:100%;height:400px;\"><\/div>",
```

Comment:
This CSS class is introduced by shiny 1.7.4 (thus the skipping if that isn't availlable)